### DISCUSSION TITLE
Unstoppable Static Dist Calls on Django server

### DISCUSSION THREAD

**Discussion by yashrajvanshi** [encode/django-rest-framework#8480] [Question & Answer]
I am not able to understand why there are repetative multiple calls coming up in logs for static files

**_admin/static/static/dist/static/dist/assets/js/static/static/dist/assets/static/dist/assets/js/php.php

/admin/static/static/dist/static/dist/assets/js/static/static/dist/assets/js/static/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/boot.ini

_/admin/static/static/dist/static/dist/assets/js/static/static/dist/assets/js/static/dist/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/%f8%80%80%80%ae%f8%80%80%80%ae/etc/pas_swd_**

These calls are repetative and making my instance slow. Did happened before a year ago and Stopped automatically. We use Angular 10 as Frontend technology couples with DRF - version -2.2.7

Can Anybody help me understand what is going wrong here

**Comment by bespokebob** [comment]
Those look like requests coming from a vulnerability scanner.

**Reply by yashrajvanshi** [reply]
Like Qualys right, but that scan was done on Friday and the calls are still coming through

**Reply by bespokebob** [reply]
Maybe your own scans are using Qualys, but if you're hosting a server on the internet, anyone can be scanning it with anything. You'd have to check your server logs for the source IP, user-agent headers, or anything else that would give you a clue. Regardless, this doesn't really have anything to do with DRF.

**Reply by yashrajvanshi** [reply]
Yup thanks, Thought of some thing related to static files or so, btw thanks for clearing this out.
